src/adcm_pytest_plugin/docker_utils.py

```diff
@@ -262,10 +262,22 @@ def run_adcm(
             port = "8000"
         else:
             container_ip = ip
-
-        wait_for_url("http://{}:{}/api/v1/".format(container_ip, port), 60)
+        self._wait_for_container_init(container, container_ip, port)
         return ADCM(container, container_ip, port)
 
+    @staticmethod
+    def _wait_for_container_init(container, container_ip, port, timeout=120):
+        if not wait_for_url(f"http://{container_ip}:{port}/api/v1/", timeout):
+            additional_message = ""
+            try:
+                container.kill()
+            except APIError:
+                additional_message = " \nWARNING: Failed to kill docker container. Try to remove it by hand"
+            raise TimeoutError(
```

Comment:
These line breaks fixture image and finalizator would not be called, am i wrong?

Reply:
Fixed